components/cloud_start_trial/cloud_start_trial_btn.scss

```diff
@@ -0,0 +1,29 @@
+.CloudStartTrialBtn {
```

Comment:
The styles of this button are temporary to match with the current learn more about trial modal steps. This will be refactored as part of the ticket https://mattermost.atlassian.net/browse/MM-43665